### PR TITLE
MISUV-486: create input and output route

### PR DESCRIPTION
**Pull request by scullythornton1**

### CODE REVIEW

File: app/repositories/SelfEmploymentsRepository.scala [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  }
+
+  def getDataFromSession(sessionId: String, dataId: String): Future[Option[JsValue]] = {
+    getSessionIdData(sessionId) map { optData =>
+      optData.flatMap { json =>
+        (json \ dataId).asOpt[JsValue]
+      }
+    }
+  }
+
```
Comment (chandraAndem) [inline app/repositories/SelfEmploymentsRepository.scala:58]: Could we add compound index on sessionId and dataId?

**Review by chandraAndem — COMMENTED** [review]
Just a minor comment on indexes. Rest all looks good.

File: app/repositories/SelfEmploymentsRepository.scala (hunk 1 above)
Comment (AlexRimmerHMRC) [reply app/repositories/SelfEmploymentsRepository.scala:58]: dataId is not a field in the document so we can't add an index on it.
We also never retrieve from the database using any dataId, just the sessionId.

Comment (chandraAndem) [reply app/repositories/SelfEmploymentsRepository.scala:58]: I was still looking it like the model I have created in my previous PR i.e., with sessionId and dataId as fields. I would assume that we don't have too many records per sessionId so its ok to live with this.

**Pull request merged**